internal/auth0/connection/schema.go

```diff
@@ -774,7 +776,7 @@ var resourceSchema = map[string]*schema.Schema{
 }
 
 func connectionSchemaV0() *schema.Resource {
-	s := resourceSchema
+	s := internalSchema.Clone(resourceSchema)
```

Comment:
Because the `resourceSchema` is a map (pointer), we were modifying the same current schema with the old schema version attributes causing the generated docs as well to display these properties that were removed in v2. We're making a clone of the resource schema instead and then safely modifying it.